src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -568,21 +569,24 @@ impl<T: AstInfo> AstDisplay for CreateSourceStatement<T> {
             f.write_str("IF NOT EXISTS ");
         }
         f.write_node(&self.name);
-        f.write_str(" ");
         if !self.col_names.is_empty() {
-            f.write_str("(");
+            f.write_str(" (");
             f.write_node(&display::comma_separated(&self.col_names));
             if self.key_constraint.is_some() {
                 f.write_str(", ");
                 f.write_node(self.key_constraint.as_ref().unwrap());
             }
-            f.write_str(") ");
+            f.write_str(")");
         } else if self.key_constraint.is_some() {
-            f.write_str("(");
+            f.write_str(" (");
             f.write_node(self.key_constraint.as_ref().unwrap());
-            f.write_str(") ")
+            f.write_str(")")
         }
-        f.write_str("FROM ");
+        if let Some(cluster) = &self.in_cluster {
+            f.write_str(" IN CLUSTER ");
```

Comment:
```suggestion
            f.write_str(" IN CLUSTER");
```

Reply:
Confused about this. We want the space between CLUSTER and the name that comes after, no?

Reply:
Ah, I didn't see that the cluster name followed.